src/main/dw/jwt/RSA.dwl

```diff
@@ -54,6 +54,11 @@ fun alg(algorithm: String) : String | Null =
 fun signJWT(jwt: String, privateKey: String, algorithm: String) : String =
     RSAHelper::signString(jwt, privateKey, algorithm)
 
+
+
+fun signJWTWithKeyStore(jwt: String, keystorePath: String, keystoreType: String, keystorePassword: String, keyAlias: String, algorithm: String) : String =
```

Comment:
Either this function needs to be inside the other function if it is internal or it should have documentation

Reply:
Added docs